packages/mermaid/src/diagrams/class/classDb.ts

```diff
@@ -456,6 +457,20 @@ export const addClassesToNamespace = function (id: string, classNames: string[])
   }
 };
 
+export const setCssStyle = function (id: string, styles: string[]) {
+  const thisClass = classes[id];
+  if (!styles || !thisClass) {
+    return;
+  }
+  for (const s of styles) {
+    if (s.includes(',')) {
+      thisClass.styles = [...thisClass.styles, ...s.split(',')];
```

Comment:
For consistency with https://github.com/mermaid-js/mermaid/blob/b92b344844256287f038c7455210ca8546f9f308/packages/mermaid/src/diagrams/class/classDb.ts#L469, we could instead use `Array.push(...items)`, which will modify the existing array in-place, instead of creating a new array.

```suggestion
      thisClass.styles.push(...s.split(','));
```